### PR TITLE
UI: Built UI and Added Routing

### PR DESCRIPTION
**Pull request by Tarandeep-s1ngh**
1. Built UI
2. Added Basic Routing

- Landing Page
- Explore
- Single Video Page
- Playlist
- Liked
- Watch Later
- History
- Login, Signup, Forgot Password, User Profile

image
image
image
image

### CODE REVIEW

File: src/components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { NavLink } from "react-router-dom";
+
+export const Sidebar = () => {
+  return (
+    <aside className="sidebar semibold">
+      <ul className="side-heading">
+        {}
```
Comment (shraddha-amber) [inline src/components/Sidebar/Sidebar.jsx]: remove these empty curly brackets

File: src/components/VideoCard/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import "./videocard.css";
+import { thumbnail1 } from "../../assets";
+import { thumbnail2 } from "../../assets";
+import { Link } from "react-router-dom";
+
+export const VideoCard = () => {
+  return (
+    <div className="card-badge">
+      <div className="card-header">
+        <img className="thumbnail-image" src={thumbnail1} alt="" />
```
Comment (shraddha-amber) [inline src/components/VideoCard/VideoCard.jsx:11]: provide some valid alt

File: src/context/product-context.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+// import { createContext, useContext, useEffect, useState } from "react";
```
Comment (shraddha-amber) [inline src/context/product-context.js]: try to avoid pushing commented code in PR

**Review by shraddha-amber — APPROVED** [review]
Very clean and beautiful UI
Liked the separated code in CSS files, and the tooltip on nav icons

Overall everything looks good

File: src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <Link to="/" className="flex-row align-items-center gap0p5">
+          <img src={sageLogo} className="img-responsive" />{" "}
+          <span>Sage Mode</span>
+        </Link>
+      </div>
+      <div className="search-bar flex-row justify-sb align-items-center">
+        <input type="text" className="search-input" />
+        <i className="fas fa-search"></i>
+      </div>
+      <Link to="/product-detail" className="icon-in-nav tooltip">
```
Comment (Nishant-96) [inline src/components/Navbar/Navbar.jsx:18]: You can use NavLink in navbars.

**Review by Nishant-96 — APPROVED** [review]
Everything looks good.
-UI is really nice.
-Good file structure.

File: src/components/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (Tarandeep-s1ngh) [reply src/components/Sidebar/Sidebar.jsx]: Done!

File: src/context/product-context.js (hunk 3 above)
Comment (Tarandeep-s1ngh) [reply src/context/product-context.js]: Removed! Pushed that by mistake, thanks for informing! ✨

**Pull request merged**